e2e-tests/support/tasks.js

```diff
@@ -1,13 +1,24 @@
 const crypto = require('node:crypto');
 const db = require('./db-client');
+const redis = require('./redis-client');
 
 module.exports = {
-  createTasks: ({ dbName, dbConnectionString }) => {
+  createTasks: ({
+    dbName,
+    dbConnectionString,
+    redisHost,
+    redisPort,
+    redisKey,
+  }) => {
     const connectionOptions = { dbName, dbConnectionString };
+    const redisConnectionOptions = { redisHost, redisPort, redisKey };
     const usersCollectionName = 'users';
+    const tfmUsersCollectionName = 'tfm-users';
```

Comment:
Should this/could this be a constant? Same with `users`

Maybe overall tech debt rather than specific to SSO